src/datomic_rest_api/rest/gene.clj

```diff
@@ -551,22 +560,126 @@
    :description
    "phenotype based on interaction"}))
         
-   
+(defn drives-overexpression [gene]
+  (let [db (d/entity-db gene)]
```

Comment:
I think this is an error. Should be:
`(d/entity db gene)`, which means you'll need to pass `db` into the function.